src/main/java/bio/terra/workspace/service/iam/model/RoleBinding.java

```diff
@@ -0,0 +1,28 @@
+package bio.terra.workspace.service.iam.model;
+
+import com.google.auto.value.AutoValue;
+import java.util.List;
+
+/** A binding between an IAM role and users/groups. */
+@AutoValue
+public abstract class RoleBinding {
+
+  /** The role granted to each of the bound users. */
+  public abstract IamRole role();
```

Comment:
Again, my wanting to understand. Does this mean that the getter is just `role()`?  Could this be renamed to `getRole()` or does it have to be the eventual name of the class member?

Reply:
Since this is an AutoValue class, the getter is just `role()` and can't be changed. 
That said AutoValue has a bit of magic where if every field is prefixed with `get_` or `is_` it knows to strip them. So you can't change them arbitrarily, but you can pick which style you want.

Reply:
Thanks for explaining.